lib/virt_autotest/utils.pm

```diff
@@ -85,19 +91,44 @@ sub restart_modular_libvirt_daemons {
     } else {
         # Restart the sockets first
         foreach my $daemon (@daemons) {
-            systemctl("restart $daemon\{,-ro,-admin\}.socket");
+            if (($daemon == 'lock') || ($daemon == 'log')) {
```

Comment:
I think there should be also ```proxy``` ?

Reply:
no, double confirm that there is the virtproxyd-ro.socket daemon for  virtproxyd. 

So, there should be covered virtlogd and virtlockd together as enough in this loop. 

FYI
```
vh016:~ # systemctl status virtproxyd-admin.socket 
○ virtproxyd-admin.socket - Libvirt proxy admin socket
     Loaded: loaded (/usr/lib/systemd/system/virtproxyd-admin.socket; disabled; vendor preset: disabled)
     Active: inactive (dead)
   Triggers: ● virtproxyd.service
     Listen: /run/libvirt/libvirt-admin-sock (Stream)
vh016:~ # systemctl status virtproxyd-ro.socket 
○ virtproxyd-ro.socket - Libvirt proxy local read-only socket
     Loaded: loaded (/usr/lib/systemd/system/virtproxyd-ro.socket; disabled; vendor preset: disabled)
     Active: inactive (dead)
   Triggers: ● virtproxyd.service
     Listen: /run/libvirt/libvirt-sock-ro (Stream)
vh016:~ # systemctl status virtproxyd.service 
○ virtproxyd.service - Virtualization daemon
     Loaded: loaded (/usr/lib/systemd/system/virtproxyd.service; disabled; vendor preset: disabled)
     Active: inactive (dead)
TriggeredBy: ○ virtproxyd-ro.socket
             ○ virtproxyd-admin.socket
             ○ virtproxyd.socket
       Docs: man:virtproxyd(8)
             https://libvirt.org
```